ui/packages/consul-ui/app/templates/dc/nodes/show/sessions.hbs

```diff
@@ -18,38 +18,97 @@ as |route|>
-          </p>
+
+        <BlockSlot @name="error" as |after error|>
+          <Consul::LockSession::Notifications
+            {{notification
+              after=(action after)
+            }}
+            @type="remove"
+            @error={{error}}
+          />
         </BlockSlot>
-        <BlockSlot @name="actions">
-          <li class="docs-link">
-            <a href="{{env 'CONSUL_DOCS_URL'}}/internals/sessions.html" rel="noopener noreferrer" target="_blank">Documentation on sessions</a>
-          </li>
-          <li class="learn-link">
-            <a href="{{env 'CONSUL_DOCS_LEARN_URL'}}/tutorials/consul/distributed-semaphore" rel="noopener noreferrer" target="_blank">Read the guide</a>
-          </li>
+
+        <BlockSlot @name="content">
+
+          <DataCollection
+            @type="session"
+            @items={{items}}
+          as |collection|>
+
+            <collection.Collection>
+              <Consul::LockSession::List
+                @items={{collection.items}}
+                @ondelete={{writer.delete}}
+              />
+            </collection.Collection>
+
+            <collection.Empty>
+              <EmptyState
+                @login={{route.model.app.login.open}}
+              >
+                <BlockSlot @name="header">
+                  <h2>
+                      Welcome to Lock Sessions
+                  </h2>
+                </BlockSlot>
+
+                <BlockSlot @name="body">
+                  <p>
+                    Consul provides a session mechanism which can be used to build distributed locks. Sessions act as a binding layer between nodes, health checks, and key/value data. There are currently no lock sessions present, or you may not have permission to view lock sessions.
+                  </p>
+                </BlockSlot>
+
+                <BlockSlot @name="actions">
+                  <li class="docs-link">
+                    <Action
+                      @href="{{env 'CONSUL_DOCS_URL'}}/internals/sessions.html"
+                      @external={{true}}
+                    >
+                      Documentation on Lock Sessions
+                    </Action>
+                  </li>
+                  <li class="learn-link">
+                    <Action
+                      @href="{{env 'CONSUL_DOCS_LEARN_URL'}}/tutorials/consul/distributed-semaphore"
+                      @external={{true}}
+                    >
+                      Read the guide
+                    </Action>
+                  </li>
+                </BlockSlot>
+
+              </EmptyState>
+            </collection.Empty>
+
+          </DataCollection>
+
         </BlockSlot>
-      </EmptyState>
-  {{/if}}
-  </div>
+      </DataWriter>
+    </div>
```

Comment:
More upgrading to new stuff, Actions, Octane, DataWriter (for deleting), DataCollection (we don't do search/sorting here but if we ever do then this is all there to be used now)